### PR TITLE
Add CVE-2017-12161

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

File: database/java/2017/12161.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    - https://bugzilla.redhat.com/show_bug.cgi?id=1484564
+    - https://github.com/keycloak/keycloak-documentation/pull/268/commits/a2b58aadee42af2c375b72e86dffc2cf23cc3770
+
+affected:
+    - groupId: org.keycloak
+      artifactId: keycloak-common
+      version:
+        - "<3.4.2 final"
+      fixedin:
+        - ">=3.4.2 final"
```
Comment (GeetikaBatra) [inline database/java/2017/12161.yaml]: Do we write final this way?

Comment (msrb) [reply database/java/2017/12161.yaml]: We take the version exactly as it is, thus in this case `3.4.2.Final` (https://mvnrepository.com/artifact/org.keycloak/keycloak-common).

**Comment by GeetikaBatra** [comment]
@msrb, PTAL

**Comment by msrb** [comment]
This CVE seems to be too fresh. Let's wait couple more days for more information to emerge.

**Pull request closed without merging**